src/postprocess/band_structure.jl

```diff
@@ -19,18 +19,30 @@ function high_symmetry_kpath(model; kline_density=20)
     # https://github.com/louisponet/DFControl.jl/blob/master/src/structure.jl
     # but for this the best way to go would be to refactor into a small "CrystalStructure"
     # julia module which deals with these sort of low-level details everyone can agree on.
-    pystructure = pymatgen_structure(model.lattice, model.atoms)
-    symm_kpath = pyimport("pymatgen.symmetry.bandstructure").HighSymmKpath(pystructure)
-    kcoords, labels = symm_kpath.get_kpoints(kline_density, coords_are_cartesian=false)
-
+    # pystructure = pymatgen_structure(model.lattice, model.atoms)
+    # symm_kpath = pyimport("pymatgen.symmetry.bandstructure").HighSymmKpath(pystructure)
+    # kcoords, labels = symm_kpath.get_kpoints(kline_density, coords_are_cartesian=false)
+    
+    sgnum = spglib_get_spacegroup(model.lattice, model.atoms)
+    Rs = [model.lattice[i, :] for i in 1:size(model.lattice,1)]
+    kp        = irrfbz_path(sgnum, Rs)
```

Comment:
can you prefix with Brillouin. so that it's clear where this comes from? For this kind of dependency it's maybe useful to not `using` but `import`, so that it's clearer where it's used